src/controller/java/src/chip/devicecontroller/ICDRegistrationInfo.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *   Copyright (c) 2020-2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package chip.devicecontroller;
+
+import javax.annotation.Nullable;
+
+public class ICDRegistrationInfo {
```

Comment:
please add doxygen description for this class regarding when and how. thanks

Reply:
Oh.... I modified.

Reply:
I added. Thanks.